test/format/config.c

```diff
@@ -275,6 +278,22 @@ config_backup(void)
         config_single(cstr, false);
     }
 }
+
+/*
+ * config_backup --
+ *     Backup configuration.
```

Comment:
Should be about "backport compatibility configuration"?

Reply:
Thanks, pushed in 94d105349.